code/modules/response_team/ert_outfits.dm

```diff
@@ -455,13 +460,19 @@
 
 /datum/outfit/job/centcom/response_team/paranormal/gamma
 	name = "RT Paranormal (Gamma)"
-	suit = /obj/item/clothing/suit/space/hardsuit/ert/paranormal/inquisitor
-	suit_store = /obj/item/gun/energy/gun/nuclear
 	l_pocket = /obj/item/grenade/clusterbuster/holy
-	shoes = /obj/item/clothing/shoes/magboots/advance
+	back = /obj/item/mod/control/pre_equipped/responsory/inquisitory/chaplain
+	gloves = null
+	shoes = null
```

Comment:
any reason for not giving them shoes at all?

Reply:
Originally, I did not port over oversuiting, and a person using a modsuit had to take off gloves and shoes to use a modsuit. As such, gamma ERT, who had a no slowdown modsuit and would not need it, should not bother spawning with these. However, we have oversuiting now, so I should add these back

Reply:
Shoes returned